Src/unstable/library/network/openssl/src/crypto/asymetric/algoritms/ssl_rsa.e

```diff
@@ -147,13 +147,58 @@ feature -- Access: Public Encrypt - Private Decrypt
 			create Result.make_from_string (l_decrypt.string)
 		end
 
-
 feature -- Sign
 
-	sign (a_priv_key: SSL_RSA_PRIVATE_KEY; a_message: READABLE_STRING_GENERAL): detachable STRING_8
+	sign_sha256 (a_priv_key: SSL_RSA_PRIVATE_KEY; a_message: READABLE_STRING_GENERAL): detachable STRING_8
```

Comment:
Not critical, but this is a bad name for a function, ... a function should not be named with a verb.

it should be
`sha256_signed_message`  or something like that.